chrome/content/zotero/xpcom/zotero.js

```diff
@@ -507,6 +509,35 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 			}
 		}
 		
+		// Check for DB purge
+		var newAccountFile = dataDir.clone();
+		newAccountFile.append('new-account');
```

Comment:
Since we'll use this for unlinking too (though that's not implemented yet, it looks like), this isn't really just for `new-account`. Can be `reset-data-directory` or something.